README.md

```diff
@@ -125,6 +129,9 @@ both Substrate chains it must be run last.
 ./deployments/local-scripts/run-rialto-bridge-node.sh
 ./deployments/local-scripts/run-millau-bridge-node.sh
 ./deployments/local-scripts/relay-millau-to-rialto.sh
+./deployments/local-scripts/relay-rialto-to-millau.sh
+./deployments/local-scripts/relay-messages-millau-to-rialto-entrypoint.sh
+./deployments/local-scripts/relay-messages-millau-to-rialto-entrypoint.sh
```

Comment:
I'd prefer if you didn't add this here.  This section specifically talks about a simple one way _header_ relay sync. This adds extra complexity, and will also not match the example shown in the Docker section below.

It does look like Andreas added a section on sending messages further down in the README, so maybe this is something that could be added there?

Reply:
well... from the UI point of view, the simple one way header relay sync is not enough since the transactions/remark  wouldn't work because it needs the message relayers. So based on this could we consider the relay messages as a part of the minimum set up to run the environment?

Reply:
For running the UI yeah I agree the setup is not enough. However, the goal of the main README is not to get somebody started on building a UI on top of the bridge, it's to get them quickly running a barebones bridge. For anything more involved we have more detailed documentation elsewhere, such as the messaging docs or the deployment README.

Reply:
ok, fair enough. I will include this in specific UI README then.